gemini/use-cases/document-rag-on-llamaindex/llamaindex_rag.ipynb

```diff
@@ -0,0 +1,1458 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "RcDLkWLmmzlB"
+      },
+      "source": [
+        "# Leverage LLamaIndex with Vertex AI Vector Search to perform question answering RAG"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "68afb8a2586f"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/noabenefraim/llamaindex_rag/blob/main/llamaindex_rag.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```

Comment:
Change this link to use the new notebook link. Also, follow the template shown in https://github.com/GoogleCloudPlatform/generative-ai/blob/main/notebook_template.ipynb

Reply:
Added.